documentation/asciidoc/computers/raspberry-pi/bootflow-2711.adoc

```diff
@@ -73,3 +73,16 @@ Please see the xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[bo
 === Bootloader Updates
 
 The bootloader may also be updated before the firmware is started if a `pieeprom.upd` file is found. Please see the xref:raspberry-pi.adoc#raspberry-pi-4-boot-eeprom[bootloader EEPROM] page for more information about bootloader updates.
+
+=== Fail-safe OS updates (TRYBOOT)
+
+The bootloader/firmware provide a one-shot flag which, if set, is cleared but causes 'tryboot.txt' to be loaded instead of 'config.txt'. This alternate firmware would specify the pending OS update firmware, cmdline, kernel and OS-prefix files. Since the flag is cleared before starting the firwmare a crash or reset will cause the original 'config.txt' file to be loaded on the next reboot.
+
+To set the `tryboot` flag add `tryboot` after the partition number in the 'reboot' command.
+```
+# Quotes are important. Reboot only accepts a single argument.
+sudo reboot `0 tryboot`
```

Comment:
Should this line use single-quotes instead of back-quotes? What's the significance of the "partition number" ?

Reply:
It has to be after the partition number which is implicitly zero